skimage/filters/tests/test_thresholding.py

```diff
@@ -304,7 +304,10 @@ def test_li_constant_image_with_nan():
 
 
 def test_li_arbitrary_start_point():
-    cell = data.cell()
+    try:
+        cell = data.cell()
+    except RuntimeError:
+        pytest.skip('pooch or git repository needed to load data.cell')
```

Comment:
Why not add pooch to our test requirements?

Reply:
(and again, then the test needs to fail gracefully when pooch can't reach the internet.)

Reply:
Why "git repository" needed?

Reply:
And why not pooch a required dep?